src/refcard/intro.md

```diff
@@ -14,69 +14,85 @@ around a binary and getting information about it.
-| f                   | List flags            |
+| f name              | Add flag "name"       |
+| f- name             | Remove flag "name"    |
+| fl                  | List flags            |
 | fd $$               | Describe an offset    |
-| fj                  | Display flags in JSON |
-| fl                  | Show flag length      |
+| fN [name]           | Show the real name    |
 | fx [flagname]       | Show hexdump of flag  |
 | fC [name] [comment] | Set flag comment      |
 
-## Flagspaces
+## Functions and variables
 
-Flags are created into a flagspace, by default none is selected,
-and listing flags will list them all. To display a subset of flags
-you can use the `fs` command to restrict it.
+Functions appear after auto-analysis or after adding them manually, e.g. with `af` command.
 
-| Command       | Description           |
-|:--------------|:----------------------|
-| fs            | Display flagspaces    |
-| fs *          | Select all flagspaces |
-| fs [space]    | Select one flagspace  |
+| Command             | Description               |
+|:--------------------|:--------------------------|
+| af                  | Analyze function          |
+| af- name            | Remove function "name"    |
+| afb                 | List basic blocks         |
+| afi                 | Show function information |
+| afs                 | Show function signature   |
+| afvl                | Show function variables   |
+
+## Global variables
+
+Global variables appear after auto-analysis or after adding them manually.
+
+| Command             | Description               |
+|:--------------------|:--------------------------|
+| avg                 | Show all globals          |
+| avg name            | Show global "name"        |
+| avga name type      | Add global variable       |
+| avgp name           | Print global variable     |
+| avgx name           | Show xrefs to the global  |
 
 ## Information
 
-Binary files have information stored inside the headers. The `i`
-command uses the RzBin api and allows us to the same things rz-bin
-do. Those are the most common ones.
+Binary files have information stored inside the headers. The `i` command uses the RzBin API and allows us to the same things rz-bin does. Those are the most common ones.
 
-| Command | Description              |
-|:--------|:-------------------------|
-| ii      | Information on imports   |
-| iI      | Info on binary           |
-| ie      | Display entrypoint       |
-| iS      | Display sections         |
-| ir      | Display relocations      |
-| iz      | List strings (izz, izzz) |
+| Command | Description                 |
+|:--------|:----------------------------|
+| ii      | Information on imports      |
+| iI      | Info on binary              |
+| ie      | Display entrypoint          |
+| iS      | Display sections            |
+| ir      | Display relocations         |
+| iz      | List strings (izz for more) |
```

Comment:
i would split this and say that `iz` is the strings found in the read only sections meanwhile `izz` are all the strings found in the raw binary

Reply:
Removed this occurrence then